modules/monitoring-configuring-monitoring-for-an-application.adoc

```diff
@@ -19,7 +19,7 @@ This procedure shows, on an example, how an application developer can deploy an
 +
 [source,yaml]
 ----
-apiVersion: extensions/v1beta1
```

Comment:
I get the following error when trying to apply this file via the UI in a 4.3 cluster:

> An error occurred
> Error "Required value" for field "spec.selector".

Reply:
We need to also update the very last thing to replace it with:

> oc port-forward -n openshift-user-workload-monitoring svc/prometheus-operated 9090

we did not include namespace.

Reply:
There was a bug in the app can you also change the image to: `quay.io/brancz/prometheus-example-app:v0.2.0`

This is my final deployment:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: example-app
  namespace: default
spec:
  selector:
    matchLabels:
      app: example-app
  replicas: 1
  template:
    metadata:
      labels:
        app: example-app
    spec:
      containers:
      - name: example-app
        image: quay.io/brancz/prometheus-example-app:v0.2.0
        ports:
        - name: web
          containerPort: 8080
---
```